index.html

```diff
@@ -4333,7 +4310,36 @@ <h2>
 
     <section>
         <h2>
-Immutability
+DIDs as URNs
+        </h2>
+
+        <p>
+If desired by a <a>DID controller</a>, a <a>DID</a> is capable of fulfilling
```

Comment:
The first of a multi-line suggestion
```suggestion
If desired by a <a>DID controller</a>, a <a>DID</a> is capable of acting
```